### DISCUSSION TITLE
Facing an Error, not able to send or receive.

### DISCUSSION THREAD

**Discussion by asimhamid** [localsend/localsend#1448] [Discussion (English, International)] [answered]
**The error:** `SocketException: Failed to create server socket (OS Error: Only one usage of each socket address (protocol/network address/ port) is normally permitted. , errno = 10048), address = 0.0.0.0, port = 53317`

I am on Windows 11 PC

**Comment by Tienisto** [comment, marked as answer]
You can configure to use a different port
